pkg/server/config.go

```diff
@@ -55,6 +57,18 @@ var (
 	DefaultGRPCConfig = GRPCConfig{
 		// No non-zero defaults yet
 	}
+
+	emptyFlagSet    = flag.NewFlagSet("", flag.ExitOnError)
+	DefaultLogLevel = func() logging.Level {
```

Comment:
Really ugly, but unfortunately I think it's indeed the quickest way of getting valid logging.Level and logging.Format values 😕

Reply:
I've written worse :)